src/marketplace/sellers/account-setup.md

```diff
@@ -6,6 +6,39 @@ redirect_from: /marketplace/sellers/account-setup-overview.html
 
 Create a new Marketplace account, or use your existing Magento account. Then, use the Marketplace Developer Portal to manage your profile information, product submissions, and the monthly payments you receive from sales on Magento Marketplace.
 
+## Bulleted list {#bulleted-list}
```

Comment:
Why has bulleted list been repeated 4 times with the headings as shown?

Reply:
Sorry for the confusion -- please ignore this PR -- it's an example for a tutorial, and I needed to take screenshots.